fabtests/pytest/sm2/conftest.py

```diff
@@ -1,11 +1,16 @@
 import pytest
 
-# TODO Restore GPU Memory Types
-@pytest.fixture(scope="module", params=["host_to_host"])
+@pytest.fixture(scope="module", params=["host_to_host",
+                                        pytest.param("host_to_cuda", marks=pytest.mark.cuda_memory),
+                                        pytest.param("cuda_to_host", marks=pytest.mark.cuda_memory),
+                                        pytest.param("cuda_to_cuda", marks=pytest.mark.cuda_memory),
+                                        pytest.param("neuron_to_neuron", marks=pytest.mark.neuron_memory),
```

Comment:
i don't think you can run sm2 with neuron traffic. libfabric does not support copy from neuron buffer to host buffer, and there is no IPC support for it.

Reply:
I copied the same fabtests that are in SHM.

Reply:
ok, that's fine then.